actors/builtin/storage_power/storage_power_actor_state.go

```diff
@@ -77,8 +73,16 @@ func (st *StoragePowerActorState) minerNominalPowerMeetsConsensusMinimum(s adt.S
 
 	var minerSizes []abi.StoragePower
 	var pwr abi.StoragePower
-	if err := adt.AsMap(s, st.PowerTable).ForEach(&pwr, func(k string) error {
-		minerSizes = append(minerSizes, pwr.Copy())
+	if err := adt.AsMap(s, st.ClaimedPower).ForEach(&pwr, func(k string) error {
+		maddr, err := addr.NewFromBytes([]byte(k))
+		if err != nil {
+			return err
+		}
+		nominalPower, err := st.computeNominalPower(s, maddr)
```

Comment:
Don't pass the address here only to redundantly look up the claimed power. `pwr` is the value just iterated: pass it as a parameter.

Reply:
Ack, computeNominalPower still needs to take in a minerAddr to check if the miner is at fault. I have made the change and will land this PR to unblock other SPA related work. Let me know if we should do this differently.